# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../heimdal/use-external-slc.patch.cross
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Use an pre-installed slc from the build host toolchain when cross-compiling

--- a/lib/hx509/Makefile.am	2014-03-13 17:10:25.884561627 +0100
+++ b/lib/hx509/Makefile.am	2014-03-13 17:10:55.037256057 +0100
@@ -147,7 +147,7 @@
 
 bin_PROGRAMS = hxtool
 
-hxtool-commands.c hxtool-commands.h: hxtool-commands.in $(SLC)
+hxtool-commands.c hxtool-commands.h: hxtool-commands.in
 	$(SLC) $(srcdir)/hxtool-commands.in
 
 dist_hxtool_SOURCES = hxtool.c
